config/initializers/catalog_inventory_listener.rb

```diff
@@ -4,6 +4,6 @@
   queue_host = ENV["QUEUE_HOST"] || "localhost"
   queue_port = ENV["QUEUE_PORT"] || 9092
 
-  topological_inventory_listener = TopologicalInventory::EventListener.new(:host => queue_host, :port => queue_port)
-  topological_inventory_listener.run
+  catalog_inventory_listener = CatalogInventory::EventListener.new(:host => queue_host, :port => queue_port)
+  catalog_inventory_listener.run
```

Comment:
@hsong-rh How about the topic name?

Reply:
topic is defined later: `platform.catalog-inventory.task-output-stream`